concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1239,8 +1239,6 @@ jobs:
             FLY_TARGET: ((deploy_env))
             FLY_CMD: ./paas-bootstrap/bin/fly
             ENABLE_GITHUB: ((enable_github))
-            GITHUB_CLIENT_ID: ((github_client_id))
-            GITHUB_CLIENT_SECRET: ((github_client_secret))
```

Comment:
With these gone they can also be removed from the `pipelines.sh` script. This in turn means that we can remove the `retrieve` action from the `manage_github_secrets.sh` script, and make it something that only handles uploading the secrets. This also means that we won't need to upload the secrets for dev deployments unless we've enabled github auth.

Reply:
I feel that it is worth keeping the retrieve in `manage_github_secrets.sh` if only for checking when pushing pipelines that the creds have been uploaded.